synapse/lib/httpapi.py

```diff
@@ -513,6 +529,31 @@ class AuthAddRoleV1(Handler):
         self.sendRestRetn(role.pack())
         return
 
+class AuthDelRoleV1(Handler):
+
+    async def post(self):
+
+        if not await self.reqAuthAdmin():
+            return
```

Comment:
should we send a resterror here?

Reply:
The reqAuthAdmin API sends one ( similar to getJsonBody )